### DISCUSSION TITLE
docs.libuv.org doesn't redirect to HTTPS

### DISCUSSION THREAD

**Discussion by okaestne** [libuv/libuv#4284] [General]
I noticed that http://docs.libuv.org does not automatically redirect from HTTP to HTTPS. Is that intentional?

**Comment by bnoordhuis** [comment]
I can't ping @nodejs/build from here but I've opened https://github.com/nodejs/build/issues/3609. Thanks for the report.

**Reply by richardlau** [reply]
This isn't controlled by Node.js' Build WG. See https://github.com/nodejs/build/issues/3609#issuecomment-1892147003.

**Reply by bnoordhuis** [reply]
Ah, okay. Fixed, at any rate.
